extension/src/plots/model/collect.test.ts

```diff
@@ -74,17 +79,24 @@ describe('collectCustomPlots', () => {
 
   it('should return checkpoint plots with values only containing selected experiments data', () => {
     const domain = customPlotsFixture.colors?.domain.slice(1) as string[]
+    const range = customPlotsFixture.colors?.range.slice(1) as Color[]
 
     const expectedOutput = customPlotsFixture.plots.map(plot => ({
       ...plot,
+      spec: isCheckpointValue(plot.type)
+        ? createCheckpointSpec(plot.metric, plot.metric, plot.param, {
```

Comment:
It was kind of complicated to create a version of the checkpoint plot for testing a change of scale since the scale is used twice in the nested object. I just used `createCheckpointSpec` since we're about to delete checkpoints plots anyway. But if prefered, it's definitely doable and I'll create a function for it!